src/lib/dnssd/Types.h

```diff
@@ -234,6 +235,9 @@ struct CommissionNodeData : public CommonResolutionData
     char instanceName[Commission::kInstanceNameMaxLength + 1] = {};
     char deviceName[kMaxDeviceNameLen + 1]                    = {};
     char pairingInstruction[kMaxPairingInstructionLen + 1]    = {};
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+    uint8_t jointFabricMode = 0;
```

Comment:
BitFlags, please.

Reply:
Addressed in the latest commit. Thanks for your feedback.